pages/instructorIssues/instructorIssues.ejs

```diff
@@ -170,9 +170,15 @@
               <% } else { %>
               <i class="fa fa-check-circle text-success issue-status-icon"></i>
               <% } %>
+              <% if (row.instance_question_id) { %>
+              <a class="d-block" href="/pl/course_instance/<%= row.course_instance_id %>/instance_question/<%= row.instance_question_id %>/?variant_id=<%= row.variant_id %>">
```

Comment:
@mwest1066 I changed the issues page to link to instance_question when possible. I believe the `urlPrefix` should not be used in this context, since you may have the Issues page from the course context instead of the instance context, can you check if that is indeed the case? Is there a better way to handle this link?